### PR TITLE
Fix: wrong calling convention for OpenGL crashing on Windows x86

### PR DESCRIPTION
**Pull request by s-m-k**
I've changed `CallingConvention.Cdecl` to `CallingConvention.Winapi`, which should fix https://github.com/MonoGame/MonoGame/issues/6962:

Docs:
> This member [Winapi] is not actually a calling convention, but instead uses the default platform calling convention. For example, on Windows the default is StdCall and on Windows CE.NET it is Cdecl.

### CODE REVIEW

**Comment by mrhelmut** [comment]
Great, I'll be testing that later today!

File: MonoGame.Framework/Platform/Graphics/OpenGL.cs [hunk 1]
```diff
@@ -557,289 +557,290 @@ internal enum RenderApi
         }
 
         internal static RenderApi BoundApi = RenderApi.GL;
+        const CallingConvention callingConvention = CallingConvention.Winapi;
```
Comment (cra0zy) [inline MonoGame.Framework/Platform/Graphics/OpenGL.cs]: Add private before const, always specify visibility.

Comment (s-m-k) [reply MonoGame.Framework/Platform/Graphics/OpenGL.cs]: Done.

**Comment by mrhelmut** [comment]
I confirm that this fixes the 32bit issue with .Net Core. Thanks!

**Pull request merged**